HW1/P2/P2b.py

```diff
@@ -1,3 +1,9 @@
 from P2 import *
 
-# Your code here
+import findspark
+findspark.init()
+
+import pyspark
+sc = pyspark.SparkContext()
+
+partitions2 = sc.parallelize(tuples).map(lambda xy: (xy, mandelbrot((xy[1]/500.0)-2,(xy[0]/500.0)-2))).partitionBy(100, lambda xy: random.randrange(0, 100, 1))
```

Comment:
I did the same thing by using the random integers. However, I divided it from 1 to 101, because I did not think there was a 0. When I had the 0, it would give me an error every once in a while.